### PR TITLE
Add functionality to see and set a user's birthdate in the admin API

### PR DESCRIPTION
**Pull request by aapeliv**
As requested by @AllhandsNoFeet 

**Backend checklist**
- [x] Formatted my code by running `autoflake -r -i --remove-all-unused-imports src && isort . && black .` in `app/backend`
- [x] Added tests for any new code or added a regression test if fixing a bug
- [ ] All tests pass
- [ ] Run the backend locally and it works
- [x] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

### CODE REVIEW

File: app/backend/src/tests/test_admin.py [hunk 1]
```diff
@@ -145,6 +149,26 @@ def test_ChangeUserGender(db):
...
+        assert res.banned == False
+        assert res.deleted == False
+
+
+def test_ChangeUserBirthdate(db):
+    with session_scope() as session:
+        _generate_normal_user(session)
+        _generate_super_user(session)
+        normal_user = _get_normal_user(session)
+
```
Comment (ShmuelTreiger) [inline app/backend/src/tests/test_admin.py:97]: Should you add a check for the user's BD before changing it?

Comment (aapeliv) [reply app/backend/src/tests/test_admin.py:97]: My thinking was that the admin is able to set it to anything, but maybe that's error prone (e.g. if they set a date in the future)

Comment (ShmuelTreiger) [reply app/backend/src/tests/test_admin.py:97]: That's completely separate, but also a good point, there should probably be a check in there (i.e. no less than 18 years old)

What I'm asking, is should it be:
-check age == default
-set age
-check age == new age

Not strictly necessary, but I think good for clarity

**Comment by aapeliv** [comment]
@ShmuelTreiger: could you do another review of this, please?

**Review by ShmuelTreiger — APPROVED** [review]
Sorry. Thought I looked at this already.

Your last commit should probably have been its own PR. I don't understand why you did it the way you did before and then changed it, but the way it is makes sense to me and I don't see any obvious bugs.

**Pull request merged**